### DISCUSSION TITLE
how to train hyp

### DISCUSSION THREAD

**Discussion by 827346462** [ultralytics/yolov5#2515] [Q&A] [answered]
when I add write '--evolve', action='store_true', help='evolve hyperparameters', default=True'' , show ''KeyError: 'anchors'''. I want train hyp,but always can't successful. please dispelling doubts.

**Comment by glenn-jocher** [comment, marked as answer]
@827346462 please uncomment the anchors hyp here and try again:
https://github.com/ultralytics/yolov5/blob/ed2c74218d6d46605cc5fa68ce9bd6ece213abe4/data/hyp.scratch.yaml#L20
